src/middleware/pretty-json/index.ts

```diff
@@ -28,13 +37,16 @@ type prettyOptions = {
  * })
  * ```
  */
-export const prettyJSON = (options: prettyOptions = { space: 2 }): MiddlewareHandler => {
+export const prettyJSON = (
+  options: PrettyOptions = { space: 2, query: 'pretty' }
```

Comment:
How about don't use the default value `{ space: 2, query: 'pretty' }`? I think this is neat.

```diff
diff --git a/src/middleware/pretty-json/index.ts b/src/middleware/pretty-json/index.ts
index 411c3e0d..bc198e58 100644
--- a/src/middleware/pretty-json/index.ts
+++ b/src/middleware/pretty-json/index.ts
@@ -37,16 +37,14 @@ interface PrettyOptions {
  * })
  * ```
  */
-export const prettyJSON = (
-  options: PrettyOptions = { space: 2, query: 'pretty' }
-): MiddlewareHandler => {
-  const targetQuery = options.query ?? 'pretty'
+export const prettyJSON = (options?: PrettyOptions): MiddlewareHandler => {
+  const targetQuery = options?.query ?? 'pretty'
   return async function prettyJSON(c, next) {
     const pretty = c.req.query(targetQuery) || c.req.query(targetQuery) === ''
     await next()
     if (pretty && c.res.headers.get('Content-Type')?.startsWith('application/json')) {
       const obj = await c.res.json()
-      c.res = new Response(JSON.stringify(obj, null, options.space ?? 2), c.res)
+      c.res = new Response(JSON.stringify(obj, null, options?.space ?? 2), c.res)
     }
   }
 }
```

Reply:
I changed it.